hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/ListOpenFilesSubCommand.java

```diff
@@ -132,7 +140,7 @@ private void printOpenKeysList(ListOpenFilesResult res) {
     if (startItem != null && !startItem.isEmpty()) {
       msg += "\nafter continuation token:\n  " + startItem;
     }
-    msg += "\n\nClient ID\t\tCreation time\tHsync'ed\tOpen File Path";
+    msg += "\n\nClient ID\t\tCreation time\tHsync'ed\tDeleted\t\tOpen File Path";
```

Comment:
If deleted is hide, we shall remove this Deleted column too.

Reply:
Done